<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTabName">
       <property name="text">
        <string>[tabname]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbAsciiMode">
       <property name="text">
        <string>ASCII Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbHexMode">
       <property name="text">
        <string>HEX mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Query:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Search results:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lvResults">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SearchDialog</sender>
   <signal>finished(int)</signal>
   <receiver>SearchDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
